tasks/lib/run.js

```diff
@@ -396,7 +418,6 @@ RunUtil.runUntilStopped = function(grunt, options, cb) {
         options.runCompleted(result, method, inputs);
       }
 
-      grunt.log.writeln();
```

Comment:
This is actually an important line, since it spaces out the outputs somewhat.

Without it, you get the following:

![screenshot from 2015-07-28 15 54 32](https://cloud.githubusercontent.com/assets/820863/8934740/f54180aa-3540-11e5-8690-f78778670722.png)

We want there to be a line break before the _Would you like to run another method?_